js_modules/dagster-ui/packages/ui-core/src/instance/backfill/BackfillPage.tsx

```diff
@@ -238,9 +251,9 @@ export const BackfillPage = () => {
                   <td>
                     <Box flex={{direction: 'row', justifyContent: 'space-between'}}>
                       <div>
-                        <Link to={assetDetailsPathForKey(asset.assetKey, linkQuery())}>
+                        <ButtonLink onClick={() => onShowAssetDetails(asset.assetKey)}>
```

Comment:
Yeah... I'm curious to see if anyone notices, but this means the links don't support cmd-click anymore. I was thinking we could pass through the event meta key state and use window.open, but we don't do that anywhere else and it seemed it might be overkill for now.